core/src/services/redis/fixtures/redis_cluster_tls/docker-compose.yml

```diff
@@ -0,0 +1,154 @@
+version: '3.8'
+
+services:
+  redis-cluster-tls-master-1:
+    image: redis:latest
+    container_name: redis-cluster-tls-master-1
+    network_mode: host
+    command: "redis-server --requirepass opendal --masterauth opendal --appendonly yes \
+      --port 0 \
+      --cluster-enabled yes \
+      --cluster-config-file nodes.conf \
+      --cluster-node-timeout 15000 \
+      --cluster-announce-ip 127.0.0.1 \
+      --cluster-announce-port 6380 \
+      --cluster-announce-bus-port 16380 \
+      --tls-protocols 'TLSv1.2 TLSv1.3' \
+      --tls-replication yes \
+      --tls-cluster yes \
+      --tls-port 6380 \
+      --tls-cert-file /etc/redis/ssl/redis.crt \
+      --tls-key-file /etc/redis/ssl/redis.key \
+      --tls-ca-cert-file /etc/redis/ssl/ca.crt \
+      --tls-auth-clients no"
+    volumes:
+      - /tmp/redis:/etc/redis/ssl/
+
+  redis-cluster-tls-master-2:
+    image: redis:latest
+    container_name: redis-cluster-tls-master-2
+    network_mode: host
+    command: "redis-server --requirepass opendal --masterauth opendal --appendonly yes \
+      --port 0 \
+      --cluster-enabled yes \
+      --cluster-config-file nodes.conf \
+      --cluster-node-timeout 15000 \
+      --cluster-announce-ip 127.0.0.1 \
+      --cluster-announce-port 6381 \
+      --cluster-announce-bus-port 16381 \
+      --tls-protocols 'TLSv1.2 TLSv1.3' \
+      --tls-replication yes \
+      --tls-cluster yes \
+      --tls-port 6381 \
+      --tls-cert-file /etc/redis/ssl/redis.crt \
+      --tls-key-file /etc/redis/ssl/redis.key \
+      --tls-ca-cert-file /etc/redis/ssl/ca.crt \
+      --tls-auth-clients no"
+    volumes:
+      - /tmp/redis:/etc/redis/ssl/
```

Comment:
This requires extra setup before we trying to start this services. Can we use `./ssl:/etc/redis/ssl/` instead?